src/components/CollapserRow.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import Collapser from './Collapser';
+import { Icon } from '@newrelic/gatsby-theme-newrelic';
+
+const collapserIcon = (isOpen) => css`
+  margin-left: auto;
+  transition: transform 0.6s ease;
+  color: var(--accent-text-color);
+  ${isOpen &&
+  `
+transform: rotate(180deg);`}
+`;
+
+const CollapserRow = ({ title, id, openByDefault, children }) => {
+  const [isOpen, toggleOpen] = useState(openByDefault);
+  const [setHeight, setHeightState] = useState(isOpen ? 'auto' : '0px');
+
+  const content = useRef(null);
+  const toggleCollapser = () => {
+    toggleOpen(!isOpen);
+    setHeightState(isOpen ? '0px' : `${content.current.scrollHeight}px`);
+  };
+
+  return (
+    <Collapser>
+      <div
+        css={css`
+          display: flex;
+          flex-direction: column;
+          border-radius: 3px;
+          margin-bottom: 8px;
```

Comment:
Seeing as this `div` is the container of the whole component, I think we should let the parent component define the space between the rows rather than have this define it. Doing so would help us use this component in more places without having to rejigger the space each time.